examples/flava/native/configs/1.8b.yaml

```diff
@@ -0,0 +1,80 @@
+training:
+  strategy: fsdp # can be changed to ddp or fsdp
```

Comment:
Is this comment true? I thought these larger ones only worked with FSDP (same question in the 2.7b config)